helmfile/values/kube-prometheus-stack-sc.yaml.gotmpl

```diff
@@ -214,11 +214,30 @@ grafana:
   grafana.ini:
     server:
       root_url: https://grafana.{{ .Values.global.opsDomain }}
+    {{ if .Values.prometheus.grafana.oidc.enabled -}}
+    auth.generic_oauth:
+      name: dex
+      enabled: {{ .Values.prometheus.grafana.oidc.enabled }}
+      client_id: grafana-ops
+      client_secret: {{ .Values.grafana.opsClientSecret }}
+      scopes: {{ .Values.prometheus.grafana.oidc.scopes }}
+      auth_url: https://dex.{{ .Values.global.baseDomain }}/auth
+      token_url: https://dex.{{ .Values.global.baseDomain }}/token
+      api_url: https://dex.{{ .Values.global.baseDomain }}/api
+      allowed_domains: {{ join " " .Values.prometheus.grafana.oidc.allowedDomains }}
+      allow_sign_up: true
+      tls_skip_verify_insecure: {{ not .Values.global.verifyTls }}
+      role_attribute_path: contains(groups[*], '{{ .Values.prometheus.grafana.oidc.userGroups.grafanaAdmin }}') && 'Admin' || contains(groups[*], '{{ .Values.prometheus.grafana.oidc.userGroups.grafanaEditor }}') && 'Editor' || contains(groups[*], '{{ .Values.prometheus.grafana.oidc.userGroups.grafanaViewer }}') && 'Viewer'
+      {{- end }}
 
   serviceMonitor:
     relabelings:
     - targetLabel: cluster
       replacement: service_cluster
+  resources: {{- toYaml .Values.prometheus.grafana.resources | nindent 4  }}
+  tolerations: {{- toYaml .Values.prometheus.grafana.tolerations | nindent 4  }}
+  affinity: {{- toYaml .Values.prometheus.grafana.affinity | nindent 4  }}
+  nodeSelector: {{- toYaml .Values.prometheus.grafana.nodeSelector | nindent 4  }}
```

Comment:
Just to make sure, will this work if you upgrade without adding the values to your config? If not mention it in the changelog or add a migration script?

Reply:
`ck8s init` is part of the migration guide, which will add these.

Reply:
With the new defaults (that comes after you run init, and thus part of the migration script) it should not make any diff except adding the request/limit. which is stated in the changelog :slightly_smiling_face:

Reply:
This is similar to the last discussion. I think the migration guide should probably be highlighted more to non-Elastisys users.